receiver/awscontainerinsightreceiver/internal/stores/localnode.go

```diff
@@ -42,18 +41,14 @@ type Decorator interface {
 }
 
 func NewLocalNodeDecorator(logger *zap.Logger, containerOrchestrator string, hostInfo hostInfo, hostName string, options ...Option) (*LocalNodeDecorator, error) {
-	nodeName := os.Getenv(ci.HostName)
-	if nodeName == "" && containerOrchestrator == ci.EKS {
-		nodeName = hostName
-		if nodeName == "" {
-			return nil, fmt.Errorf("missing environment variable %s. Please check your deployment YAML config or agent config", ci.HostName)
-		}
+	if hostName == "" && containerOrchestrator == ci.EKS {
```

Comment:
I assume if we have a node name it means we have to have a host name

Reply:
Yes. The host name is used as the node name in this case.